pandas/core/arrays/datetimes.py

```diff
@@ -1363,6 +1383,14 @@ def timetz(self) -> npt.NDArray[np.object_]:
         0    10:00:00+00:00
         1    11:00:00+00:00
         dtype: object
+
+        For DatetimeIndex:
+
+        >>> idx = pd.DatetimeIndex(["1/1/2020 10:00:00+00:00",
+        ...                         "2/1/2020 11:00:00+00:00"])
+        >>> idx.timetz
+        array([datetime.time(10, 0, tzinfo=datetime.timezone.utc),
```

Comment:
this looks correct, but I'm genuinely confused as to why anyone would want to attach a time zone to a time - don't time zones only make sense if there's both a date and a time component? anyway, this is unrelated to your PR :smile: